mne/datasets/eegbci/eegbci.py

```diff
@@ -90,8 +83,8 @@ def load_data(subject, runs, path=None, force_update=False, update_path=None,
               base_url=EEGMI_URL, verbose=None):  # noqa: D301
     """Get paths to local copies of EEGBCI dataset files.
 
-    This will fetch data for the EEGBCI dataset [1]_, which is also
-    available at PhysioNet [2]_.
+    This will fetch data for the EEGBCI dataset :footcite:'SchalkEtAl2004', which is also
+    available at PhysioNet :footcite:'GoldbergerEtAl2000'.
```

Comment:
```suggestion
    This will fetch data for the EEGBCI dataset :footcite:`SchalkEtAl2004`, which is also
    available at PhysioNet :footcite:`GoldbergerEtAl2000`.
```